arviz/stats/stats.py

```diff
@@ -1259,6 +1262,10 @@ def summary(
     if not isinstance(kind, str) or kind not in kind_group:
         raise TypeError(f"Invalid kind: '{kind}'. Kind options are: {kind_group}")
 
+    focus_group = ("mean", "median")
+    if not isinstance(stat_focus, str) or (stat_focus not in focus_group):
+        raise TypeError(f"Invalid format: '{stat_focus}'. Focus options are: {focus_group}")
```

Comment:
We could make stat_focus default to `None` and when this happens take the default from rcParams, not necessarly in this PR but trying to discuss and agree on an api. @canyon289?

Reply:
Current implementation of `stat_focus` would only allow `"mean"` and `"median"` to be passed to `stat_focus`. If nothing is passed then the summary returns the default output that is it returns mean focused summary ( therefore I set the default to `"mean"`). And if any other input is passed to `stat_focus` then `az.summary` will produce this Type Error.
Should I change it to `None`? Also should I modify the error handling part here? 

Also what should be the desired output if we provide a dictionary in `stat_funcs` in addition to providing a string to `stat_focus`?